node/src/chain_spec/turing.rs

```diff
@@ -146,6 +153,22 @@ pub fn turing_development_config() -> ChainSpec {
 							},
 						),
 					),
+					(
+						4,
+						orml_asset_registry::AssetMetadata::<Balance, CustomMetadata>::encode(
+							&orml_asset_registry::AssetMetadata {
+								decimals: 18,
+								name: b"Shibuya".to_vec(),
+								symbol: b"SBY".to_vec(),
+								existential_deposit: 10_000_000_000_000_000,
+								location: Some(MultiLocation::new(1, X1(Parachain(2000))).into()),
+								additional: CustomMetadata {
+									fee_per_second: Some(416_000_000_000),
+									conversion_rate: None,
+								},
+							},
+						),
+					),
```

Comment:
I supposed that this is a dev config. Do we want to add it?

Reply:
Since this is the turing_development_config it won't break anything by being added here, but it seems like we've added atleast some unnecessary things. Do we really need Rocstar, Shiden, and Shibuya in the dev config? Presumably only one of those will be used for dev.

Reply:
@imstar15 this is dev config. Do we really need RSTAR and SDN registration here?